src/Mod/Fem/femtest/app/support_utils.py

```diff
@@ -228,17 +228,21 @@ def compare_files(
     file1 = open(file_name1, "r")
     f1 = file1.readlines()
     file1.close()
+
+    # TODO: add support for variable values in the reference file
```

Comment:
I don't understand this comment. What is missing and why can't it be added right now?

Reply:
I just added it as a suggestion if someone wants to improve the test case in the future. The current test case just removes any lines with variables in them ('Merge' and 'Save'), and as NumThreads is also a variable that is removed too.